recipes/rte-rrtmgp/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "rte-rrtmgp" %}
+{% set version = "0.0.1" %}
+{% set run_validation_plots = false %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # TODO: Set `makepath-cmake` to {{ version }} when we have real version
```

Comment:
You have to address this or we can't publish it. Either use `v1.9-alpha1` above or ask upstream to release `0.0.1`, but the versions must match.